content/en/contribute/_index.md

```diff
@@ -7,111 +7,127 @@ menu:
     weight: 10
 ---
 
-The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
-where you'll find a list of [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls), and 
-[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
+## Welcome 
 
-There are four ways you can contribute:
+Welcome to the Cloud Native Glossary contributing guide and thank you for your interest. If you would like to contribute to our project, checkout the following list of contribution types and their corresponding sections in this guide: 
 
 1) [Work on an existing issue](#work-on-an-existing-issue)
 2) [Propose new terms](#propose-new-terms)
 3) [Update existing ones](#update-an-existing-term)
-4) [Help translate the glossary](#help-translate-the-glossary)
+4) [Localize the glossary](#help-translate-the-glossary)
+
+## CNCF glossary overview 
+
+The goal of this glossary is to simplify the cloud native space — which is notorious for its complexity — and thus make it more accessible to people. 
+
+The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
+where you'll find a list of [issues](https://github.com/cncf/glossary/issues), pull requests ([PRs](https://github.com/cncf/glossary/pulls)), and 
+[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space. 
```

Comment:
```suggestion
How you can participate in this project depends on your level of cloud native expertise. 
```
I really want to stay away from "anyone can contribute." First of all, that is not true. We've seen a bunch of contributions from people who do not have the qualification to do so. While localization is an option, it isn't for people who only are proficient in English. 

We introduced this section because of the false belief that everyone can participate. Without the proper know-how, it creates a lot of frustration for contributors and maintainers. I want us to be very transparent here to avoid that. Currently, I'm pointing everyone who wants to contribute a new term to this section and have them confirm that they qualify. It may sound harsh, but we need to set clear expectations.